src/Razor/src/Microsoft.VisualStudio.Editor.Razor/DefaultDynamicDocumentContainer.cs

```diff
@@ -49,21 +47,15 @@ public override IRazorDocumentExcerptServiceImplementation GetExcerptService()
 
         public override IRazorSpanMappingService GetMappingService()
         {
-            if (_mappingService is null)
-            {
-                _mappingService = new RazorSpanMappingService(_documentSnapshot);
-            }
+            _mappingService ??= new RazorSpanMappingService(_documentSnapshot);
 
             return _mappingService;
         }
 
-        public override IRazorDocumentPropertiesService GetDocumentPropertiesService()
-        {
-            // DocumentPropertiesServices are used to tell Roslyn to provide C# diagnostics for LSP provided documents to be shown
-            // in the editor given a specific Language Server Client. Given this type is a container for DocumentSnapshots, we don't
-            // have a Language Server to associate errors with or an open document to display those errors on. We return `null` to
-            // opt out of those features.
-            return null;
-        }
+        // DocumentPropertiesServices are used to tell Roslyn to provide C# diagnostics for LSP provided documents to be shown
+        // in the editor given a specific Language Server Client. Given this type is a container for DocumentSnapshots, we don't
+        // have a Language Server to associate errors with or an open document to display those errors on. We return `null` to
+        // opt out of those features.
+        public override IRazorDocumentPropertiesService? GetDocumentPropertiesService() => null;
```

Comment:
Very much preferred the previous code here. If the comment above a method doesn't make sense to be a doc comment, then it shouldn't be above the method, IMO

Reply:
Yes, yes, I got it...